README.md

```diff
@@ -89,3 +89,7 @@ email to acknowledge your report, and we'll send an additional email when we've
 positively or negatively.
 
 For further details please see our complete [security release process](SECURITY.md).
+
+## Releases
+
+For furth details please see our [release process](https://github.com/envoyproxy/envoy/blob/main/RELEASES.md).
```

Comment:
s/furth/further/

Reply:
Doh. 🤦. Fixed in https://github.com/envoyproxy/envoy/pull/35264/commits/c522a8d1b557dd149ce1d5df7e981ee0566d58bf.